wordpress-mobile/WordPress-iOS
# Gutenberg Mentions: Allow editing of existing mentions

Fixes #

This PR adds the possibility to edit an existing mention

To test:
 - Start a new post in a P2 site
 - Add a mention to the content of a block
 - Move the cursor/caret over a location in a mention
 - Tap on the @ button in the toolbar 
 - Check that the mention UI shows up prefilled with the current value of the mention
 - Select another value for the mention
 - See if the new mention replaces the previous mention
 - Move the cursor again somewhere inside the mention
 - This time tap backspace on the keyboard
 - Check that the mention UI shows up and you can change the existing mention like explained above.


PR submission checklist:
